solara/components/file_drop.py

```diff
@@ -25,25 +26,51 @@ class FileDropZone(FileInput):
     template_file = (__file__, "file_drop.vue")
     items = traitlets.List(default_value=[]).tag(sync=True)
     label = traitlets.Unicode().tag(sync=True)
+    multiple = traitlets.Bool(True).tag(sync=True)
+
+
+@overload
+@solara.component
+def FileDrop(
+    label: str = ...,
+    on_total_progress: Optional[Callable[[float], None]] = ...,
+    on_file: Optional[Callable[[FileInfo], None]] = None,
+    lazy: bool = ...,
+    multiple: Literal[False] = ...,
+) -> ipyvuetify.extra.FileInput:
+    ...
+
+
+@overload
+@solara.component
+def FileDrop(
+    label: str = ...,
+    on_total_progress: Optional[Callable[[float], None]] = ...,
+    on_file: Optional[Callable[[List[FileInfo]], None]] = None,
+    lazy: bool = ...,
+    multiple: Literal[True] = ...,
+) -> ipyvuetify.extra.FileInput:
+    ...
 
```

Comment:
Hi @hkayabilisim! Again, thanks for the contribution. 

The remaining issue is due to the how the component is used in the documentation: `multiple_upload` is a variable, and therefore not a Literal, so it doesn't match any of the overload variants. This can be fixed by adding an overload variant which takes a `bool` instead of `Literal[...]` for `multiple`, i.e. 

```suggestion
@overload
@solara.component
def FileDrop(
    label=...,
    on_total_progress: Optional[Callable[[float], None]] = ...,
    on_file: Optional[Callable[[Union[FileInfo, List[FileInfo]]], None]] = ...,
    lazy: bool = ...,
    multiple: bool = ...,
) -> ipyvuetify.extra.FileInput:
    ...
```